app/views/idv/phone/new.html.erb

```diff
@@ -19,7 +19,7 @@
 
 <%= render(VendorOutageAlertComponent.new(vendors: [:sms, :voice], context: :idv)) %>
 
-<%= render PageHeadingComponent.new.with_content(t('idv.titles.session.phone')) %>
+<%= render PageHeadingComponent.new.with_content(t('titles.idv.phone')) %>
```

Comment:
Are there english and french versions of this title? I only see `titles/es.yml` in this pr

Reply:
yeah it's the only one that had content changes

Reply:
ah, I see, for some reason I missed the `titles.idv.phone` on my local checkout.